src/Admin/PageAdmin.php

```diff
@@ -175,7 +175,13 @@ protected function configureListFields(ListMapper $list): void
             ])
             ->add('decorate', null, ['editable' => true])
             ->add('enabled', null, ['editable' => true])
-            ->add('edited', null, ['editable' => true]);
+            ->add('edited', null, ['editable' => true])
+            ->add(ListMapper::NAME_ACTIONS, ListMapper::TYPE_ACTIONS, [
+                'translation_domain' => false,
```

Comment:
I am not sure how to let sonata know I do not want SonataPageBundle as translation domain for this specific field, otherwise this happens:

![image](https://user-images.githubusercontent.com/1137485/189295472-7b627e25-22f5-41f9-a4a4-672e38293ef1.png)

Reply:
Translation false wont translate. Shouldnt it be SonataAdminBundle ?

Reply:
just tried, it doesn't matter.

Reply:
It does get picked up as SonataAdminBundle on the translation domain, but there isn't any translation for it on the xliff files of SonataAdminBundle ....

Reply:
add a edit button a good approach for this links issues!